docs/developer.rst

```diff
@@ -184,7 +184,7 @@ A full blown example would look like this (needs to be utf-8 encoded):
         <website>https://github.com/owncloud/news</website>
         <bugs>https://github.com/owncloud/news/issues</bugs>
         <repository>https://github.com/owncloud/news</repository>
-        <screenshot>https://example.com/1.png</screenshot>
+        <screenshot small-thumbnail="https://example.com/1-small.png">https://example.com/1.png</screenshot>
```

Comment:
So I always have to add a non-small thumbnail as well in addition to the small one? Seems kinda odd considering the API at https://github.com/nextcloud/appstore/pull/383/files#diff-35e3932036a7062143635da00f64ccbdR319